src/bin/get_geneInfo.pl

```diff
@@ -118,17 +117,15 @@
 		my $tree = $gm->sub_model($full_chr, $strand);
 		next unless($tree);
 		my @tmp = $tree->intersect([$start, $end]);
-		#print "number of genes: ", scalar @tmp, " at ", $full_chr, " ", $strand, "\n";
 		foreach my $tnode (@tmp) {
 			my $g=$tnode->val;
-			#$gRange =~ s/chr//;
 			my $mRNA_length = $g->get_mRNA_length;
 			my $gRange = $chr.":".$g->start."-".$g->end;
 			my $cnt = 0;
 			$sam->fetch($gRange, 
 				sub {
 					my $a = shift;
-					return if( ($a->flag & 0x0400) || ($a->flag & 0x0004) ); #PCR dumplicate or unmapped
```

Comment:
Dumplicate sounds like an awesome name for dumplings that come in pairs.

Reply:
Unfortunately, this is not the first time I've made this change, it was overwritten repeatedly in the svn version.